### PR TITLE
[NEW] Using the Linode CLI

### PR DESCRIPTION
**Pull request by Jskobos**

### CODE REVIEW

File: docs/platform/api/using-the-linode-cli.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+
+You can view or update your account information, add payment methods, view notifications, make payments, create OAuth clients, and more through the `account` action.
+
+1.  View your account:
+
+        linode-cli account view
+
+2.  View all of your OAuth clients:
+
+        linode-cli account clients-list
```
Comment (Dorthu) [inline docs/platform/api/using-the-linode-cli.md]: This is a really fringe feature - I might not plug it here.  OAuth Clients are only useful to developers making applications that people will log into with their Linode account, [like this one](https://rnd.dorthu.com/linode-chart.html).  Most people probably won't do this.

File: docs/platform/api/using-the-linode-cli.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+3.  View your account settings:
+
+        linode-cli account settings
+
+4.  Make a payment:
+
+        linode-cli account payment-create --cvv 123 --usd 20.00
```
Comment (Dorthu) [inline docs/platform/api/using-the-linode-cli.md]: You should include an example of viewing events and notifications, as those are important and will likely be a common usage of the CLI. 

You might also include some examples of profile management through the CLI, such as viewing/deleting tokens and authorized apps.

Tickets are probably a popular feature to have in here, but they don't display well right now 😕  too much text wrapping.  Might be worth mentioning anyway.

File: docs/platform/api/using-the-linode-cli.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+    linode-cli types list
+    linode-cli images list
+
+### Linodes
+
+Tasks related to Linode instances are performed with `linode-cli linodes [ACTION]`.
+
+1.  List all of the Linodes on your account:
+
+        linode-cli linodes list
```
Comment (Dorthu) [inline docs/platform/api/using-the-linode-cli.md]: You could include examples of filtering here.  For example, `linode-cli linodes list --region us-east` to see all Linodes in one region.  The `--help` tells you what columns are filterable.

File: docs/platform/api/using-the-linode-cli.md [hunk 4]
```diff
@@ -0,0 +1,318 @@
...
+
+    You can filter results to a particular region:
+
+        linode-cli linodes list --region us-east
+
+    Filtering works on many fields throughout the CLI; use `--help` for each action to see which properties are filterable.
+
+2.  Create a new Linode:
+
+        linode-cli linodes create --label new-linode --root_pass mypassword
```
[the remark this answers is not in the document]
Comment (cwlinode) [reply docs/platform/api/using-the-linode-cli.md]: Also:
```
user@localhost:~$ linode-cli linodes create --help
linode-cli linodes create
Create Linode

Arguments:
Traceback (most recent call last):
  File "/usr/local/bin/linode-cli", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/linodecli/__init__.py", line 193, in main
    for arg in sorted(operation.args, key=lambda s: not s.required):
  File "/usr/local/lib/python2.7/dist-packages/linodecli/__init__.py", line 193, in <lambda>
    for arg in sorted(operation.args, key=lambda s: not s.required):
AttributeError: CLIArg instance has no attribute 'required'
```

**Pull request merged**